base/applications/mspaint/canvas.h

```diff
@@ -55,7 +56,8 @@ class CCanvasWindow : public CWindowImpl<CCanvasWindow>
     CANVAS_HITTEST m_hitSelection;
     CANVAS_HITTEST m_whereHit;
     POINT m_ptOrig; // The origin of drag start
-    HBITMAP m_hbmCached; // The cached buffer bitmap
+    HBITMAP m_hbmCached1; // The cached buffer bitmap #1
+    HBITMAP m_hbmCached2; // The cached buffer bitmap #2
```

Comment:
hmmm... or array: `HBITMAP m_hbmCached[2];` ?

Reply:
Renamed and make them an array as `m_ahbmCached[2]`.